### PR TITLE
XD-3570 Documentation update for Cloud Foundry Module

### PR DESCRIPTION
**Pull request by twoseat**
This change updates the documentation in the overall project readme and
the asciidoc to reflect the latest changes in the code for the Cloud
Foundry Module

### CODE REVIEW

**Comment by ericbottard** [comment]
Paul, could you please rebase this PR on top of the latest master?

**Comment by twoseat** [comment]
Done.

File: spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -43,6 +47,7 @@ Build the project:
 === Deploying 'local'
 
 [start=1]
+<<<<<<< HEAD
```
Comment (ericbottard) [inline spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc]: Ooops :)
Will remove

**Comment by ericbottard** [comment]
Merged as bf2459f.

**Pull request closed without merging**